public/index.js

```diff
@@ -0,0 +1,77 @@
+// The count is used to give each movie an id number so they can be targetted later on. The first movie in the list will have id=1 since the count is ++ before movie is appended.
+// newId is used to add the same id number as the count to the movie's object so it can be targetted later on.
+var count = 0
+var newId = 0
+
+$(document).ready(function () {
+
+  //GETTING ALL THE MOVIES
+  $.get('/movies', function (data) {
+    for (var i=0;i<data.length;i++) {
+
+      count++
+      newId = data.length
+
+      $('.movies').append(
+        '<tr id=' + count + '>' +
```

Comment:
Template strings that use backticks actually can be multi-line without using `+`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Multi-line_strings